commit 5cb23ac98046a07d5db3a52b41e3a42544eb13f9
Author: Jan Lepper <jan_lepper@gmx.de>
Date:   Wed Jan 2 21:38:52 2013 +0100

    FIXED: [ 309119 ] Wrong "New directory" dialog behavior: can't clear all typed text
    return QValidator::Intermediate instead of Invalid when input is empty
    BUG: 309119

diff --git a/krusader/Panel/panelfunc.cpp b/krusader/Panel/panelfunc.cpp
index 2f696da..8935ddf 100644
--- a/krusader/Panel/panelfunc.cpp
+++ b/krusader/Panel/panelfunc.cpp
@@ -100,7 +100,7 @@ class FilePathValidator : public QValidator
 {
 public:
     virtual State validate(QString &input, int &pos) const {
-        return input.isEmpty() ? Invalid : Acceptable;
+            return input.isEmpty() ? Intermediate : Acceptable;
     }
 };
 
